images/Dockerfile.alpine-arm-rpi

```diff
@@ -3,45 +3,37 @@ ARG BASE_IMAGE=alpine
 FROM $BASE_IMAGE
 
 RUN apk --no-cache add  \
-      grub \
-      grub-efi \
       bash \
-      connman \
-      gettext \
-      squashfs-tools \
-      openrc \
-      parted \
-      e2fsprogs \
-      dosfstools \
-      coreutils \
-      which \
-      logrotate \
-      curl \
-      nano \
-      gawk \
-      haveged \
-      tar \
-      rsync \
       bash-completion \
       blkid \
       busybox-openrc \
       ca-certificates \
+      connman \
       conntrack-tools \
       coreutils \
+      coreutils \
       cryptsetup \
       curl \
+      curl \
       dbus \
       dmidecode \
       dosfstools \
+      dosfstools \
+      e2fsprogs \
```

Comment:
so do these ... will not overflow with more comments but I guess is the same in other files